### ISSUE TITLE
Write parser for perfmon json files

### ISSUE THREAD

**Issue by cmcantalupo** [geopm/geopm#138]
The perfmon web site supports descriptions of programmable counters here:

https://download.01.org/perfmon/

Introduce a parser that can digest the JSON files provided there and enable PlatformIO to program on core events.  Support for off core counters is a stretch goal.

**Comment by dianarg** [comment]
We need more details about the details needed for programming counters before this task can be completed. 
Depends on #123.
